src/modules/new-request-form/translations/en-us.yml

```
@@ -0,0 +1,134 @@
+title: "Copenhagen Theme New Request Form"
+packages:
+  - "new-request-form"
+parts:
+  - translation:
+      key: "new-request-form.answer-bot-modal.title"
+      title: "Title for the Answer Bot modal"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "While you wait, do any of these articles answer your question?"
+  - translation:
+      key: "new-request-form.answer-bot-modal.view-article"
+      title: "Label for the link to view a suggested article in the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "View article"
+  - translation:
+      key: "new-request-form.answer-bot-modal.footer-title"
+      title: "Title of the footer of the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "Does this article answer your question?"
+  - translation:
+      key: "new-request-form.answer-bot-modal.footer-content"
+      title: "Title of the footer of the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "If it does, we can close your recent request #{{requestId}}"
+  - translation:
+      key: "new-request-form.answer-bot-modal.mark-irrelevant"
+      title: "Label of the button used to mark a suggested article as irrelevant in the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "No, I need help"
+  - translation:
+      key: "new-request-form.answer-bot-modal.solve-request"
+      title: "Label of the button used to close the request in the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "Yes, close my request"
+  - translation:
+      key: "new-request-form.answer-bot-modal.request-submitted"
+      title: "Notification used in the Answer Bot Modal to inform the user that their request was submitted"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "Your request was successfully submitted."
+  - translation:
+      key: "new-request-form.answer-bot-modal.request-closed"
+      title: "Notification shown after a request has been closed from the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/1ypC8cFgu198OkiUkr9I0cqtzIjqZl9PQ/view?usp=drive_link"
+      value: "Nice! Your request has been closed."
+  - translation:
+      key: "new-request-form.answer-bot-modal.solve-error"
+      title: "Notification shown when there is an error closing a request from the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/1AJxNZ6HNoFbANJiRSfzOxUTGfJ2XYqyH/view?usp=sharing"
+      value: "Oops! There was an error closing your request."
```

```suggestion
      value: "There was an error closing your request"
```
Content design doesn't want us to use words like "oops": https://garden.zendesk.com/content/word-list#oops-whoops-oh-no
Removing the period since there's only one sentence in the suggestion :)